README.rst

```diff
@@ -260,3 +260,10 @@ To get statement coverage::
 
     $ coverage run --source edx_sga manage.py lms --settings=test test edx_sga
     $ coverage report -m
+
+To run JavaScript tests, install js-test-tool_ and run::
+
+    $ js-test-tool run ./edx_sga/static/js_test.yml --use-firefox
```

Comment:
Do we need to specify that user should be location below to run test (Devstack) ?
> /edx/app/edxapp/venvs/edxapp/src/edx-sga